src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task {
+
+    public Event(String description) {
+        super(description);
+    }
+
+    @Override
+    public String getType() {
+        return ("E");
+    }
+
+    @Override
+    public String getDescription() {
```

Comment:
Perhaps you can comment what is the expected format of the string this method returns